src/main/java/com/imaee/propinq/neighborhood/controllers/implementations/NeighborhoodController.java

```diff
@@ -0,0 +1,43 @@
+package com.imaee.propinq.neighborhood.controllers.implementations;
+
+import com.imaee.propinq.neighborhood.controllers.interfaces.INeighborhoodController;
+import com.imaee.propinq.neighborhood.controllers.requests.NeighborhoodRequest;
+import com.imaee.propinq.neighborhood.controllers.responses.NeighborhoodResponse;
+import com.imaee.propinq.neighborhood.services.interfaces.INeighborhoodService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+public class NeighborhoodController implements INeighborhoodController {
+    @Autowired
+    private INeighborhoodService neighborhoodService;
```

Comment:
Misma corrección que el controller anterior

```suggestion
@RestController
@AllArgsConstructor
public class NeighborhoodController implements INeighborhoodController {
    
    private final INeighborhoodService neighborhoodService;
```